spec/helpers/record_helper_spec.rb

```diff
@@ -114,24 +126,28 @@
           )
         end
       end
-      it 'should join the subject fields in a dd' do
-        expect(subject).to match /<dd><a href=*.*\">Subject1a*.*Subject1b<\/a><\/dd>\s*<dd><a/
+
+      it 'joins the subject fields in a dd' do
+        expect(subject).to match(%r{<dd><a href=*.*">Subject1a*.*Subject1b</a></dd>\s*<dd><a})
       end
-      it "should join the individual subjects with a '>'" do
-        expect(subject).to match /Subject2b<\/a> &gt; <a href/
+
+      it "joins the individual subjects with a '>'" do
+        expect(subject).to match(%r{Subject2b</a> &gt; <a href})
       end
-      it 'should not print empty labels' do
+
+      it 'does not print empty labels' do
         expect(mods_subject_field(empty_field)).not_to be_present
       end
     end
 
     describe '#mods_genre_field' do
-      it 'should join the genre fields with a dd' do
+      it 'joins the genre fields with a dd' do
         expect(mods_genre_field(genres.first) do |text|
           link_to(text, searches_path(q: text))
-        end).to match /<dd><a href=*.*>Genre1*.*<\/a><\/dd>\s*<dd><a*.*Genre2<\/a><\/dd>/
+        end).to match(%r{<dd><a href=*.*>Genre1*.*</a></dd>\s*<dd><a*.*Genre2</a></dd>})
```

Comment:
`%r{}` is new to me! neat.

Reply:
huh, looks like it removes the need to escape things? i wonder if it's like python's "raw string" syntax `r"foo"`. thanks, rubocop!

Reply:
`%r{}` == `//` except you don't have to escape any `/` in your regex.